hs_core/management/commands/repair_resource.py

```diff
@@ -67,13 +71,18 @@ def handle(self, *args, **options):
             print("FILTERING TO INCLUDE PUBLISHED RESOURCES ONLY")
             resources = resources.filter(raccess__published=True)
 
-        if days:
-            print(f"FILTERING TO INCLUDE RESOURCES UPDATED IN LAST {days} DAYS")
+        if updated_since:
+            print(f"FILTERING TO INCLUDE RESOURCES UPDATED IN LAST {updated_since} DAYS")
             if resources_ids:
-                print("Your supplied resource_ids will be filtered by the --days that you provided. ")
-            cuttoff_time = timezone.now() - timedelta(days)
+                print("Your supplied resource_ids will be filtered by the --updated_since days that you provided. ")
```

Comment:
Do we want to do a similar print() inside the line 81 if block?

Reply:
doesn't hurt! added